src/App/Pathfinder/Scan.hs

```diff
@@ -133,8 +134,8 @@ instance ToJSON CompletedLicenseScan where
 mkLicenseScan :: DiscoveredProject n -> [LicenseResult] -> ProjectLicenseScan
 mkLicenseScan project licenses =
   ProjectLicenseScan
-    { licenseStrategyType = projectType project
-    , licenseStrategyName = projectType project
+    { licenseStrategyType = toText (show $ projectType project)
+    , licenseStrategyName = toText (show $ projectType project)
```

Comment:
If we keep the `ToText` instance, this should just be `toText`.  Otherwise, we can use `showT` here.